src/namespace/privilegednamespace.js

```diff
@@ -287,4 +362,165 @@ export class PrivilegedNamespace extends CustomNamespace {
         callback([allResponses[0], allResponses[1], allResponses[2]]);
 
     };
+
+    /**
+     * Ban a user from all admin future debates and kick him immediately if debateId is specified
+     * banObj contains the required information (uuid and debateId)
+     */
+    banUser = (socket) => async (banObj, callback) => {
+        logger.debug(`banUser received from user (${socket.username}), id(${socket.id})`);
+
+        if (!TypeCheck.isFunction(callback)) {
+            logger.debug(`callback is not a function.`);
+            return;
+        }
+
+        let {uuid, debateId} = banObj;
+        if (!TypeCheck.isString(uuid) || !TypeCheck.isInteger(debateId)) {
```

Comment:
If `debateId` is not specified `TypeCheck.isInteger(debateId)` will probably enter the if statement and return with `callback(false)`.

Reply:
Fixed it in 0e6bb7b